test-dist/testnew.go

```diff
@@ -267,7 +267,7 @@ func versionMatch(a, b string) bool {
 		return af == bf
 	}
 
-	return a != b
```

Comment:
the condition was reverse for some reason.

See: https://github.com/ipfs/ipfs-update/pull/51/commits/10d14bf7d320a3c64c47c9ba181ac7ab4f27eda2#diff-7137c645a23d1f7352d1942f239feb23R219